rclcpp/cmake/rclcpp_create_node_main.cmake

```diff
@@ -0,0 +1,28 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
It was copied... the original file is still there. You can do whatever you want though.